src/listen.js

```diff
@@ -160,6 +160,11 @@ module.exports = function(defaultFuncs, api, ctx) {
 
               return globalCallback(null, fmtMsg);
               break;
+            case 'inbox':
+              // Could be message request
+              // Have to use getMessageRequests() for more infomation.
+              return globalCallback(null, v);
```

Comment:
As you can see in the rest of this file, we try to avoid returning raw facebook objects whenever possible.  They have a habit of changing without warning and breaking the signature of the api.